components/src/AppHeader/appheader.main.tsx

```diff
@@ -255,7 +256,11 @@ class AppHeaderMain extends React.Component<AppHeaderMainProps, AppHeaderMainSta
 
           <div className="icons-header-container">
             <div className="search-container">
-              <AppHeaderSearchMain isMobileView={false} onSearchPage={onSearchPage} />
+              {Config.bloomreachSearch.enable ? (
+                <BloomreachAppHeaderSearchMain isMobileView={false} onSearchPage={onSearchPage} />
+              ) : (
+                <AppHeaderSearchMain isMobileView={false} onSearchPage={onSearchPage} />
```

Comment:
Just to confirm, we're still using EP search on mobile then?

Reply:
This is just one section of the implementation.  I haven't yet included bloomreach search in any of my PR's.  Just search suggestion.  Will include bloomreach search in following PR's

Reply:
@aChanEP Should this also be included on mobile though? `AppHeaderSearchMain` is imported on mobile below here